src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    private Ball ball = new Ball();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
Random should be a class level variable - you should not create it each time this method is invoked